comments-bundle/src/ContaoManager/Plugin.php

```diff
@@ -27,8 +30,12 @@ public function getBundles(ParserInterface $parser): array
     {
         return [
             BundleConfig::create(ContaoCommentsBundle::class)
-                ->setLoadAfter([ContaoCoreBundle::class])
-                ->setReplace(['comments']),
+                ->setLoadAfter([
+                    ContaoCoreBundle::class,
+                    ContaoCalendarBundle::class,
+                    ContaoFaqBundle::class,
+                    ContaoNewsBundle::class],
+                )->setReplace(['comments']),
```

Comment:
```suggestion
                    ContaoNewsBundle::class,
                ])->setReplace(['comments']),
```

Code style checker would probably mention this?